### DISCUSSION TITLE
Is there a way to move the year dropdown?

### DISCUSSION THREAD

**Discussion by siminn-elisajo** [Hacker0x01/react-datepicker#3513] [Q&A]
Hi,

I want to include a year dropdown but I want to display it next to the month name, where the current year is displayed. Is this possible?

**Comment by jktravis-cps** [comment]
I, too, would like to move the month year dropdowns. Specifically, I'd like to remove the duplicated header that contains those values and instead make those be the dropdowns instead. 

<img width="273" alt="image" src="https://github.com/user-attachments/assets/936cb248-6d07-4f2d-8614-685cb3bb8883" />

End result might look like this

<img width="259" alt="image" src="https://github.com/user-attachments/assets/b153743d-f69a-4bd0-bb20-555eb51734d8" />
